### ISSUE TITLE
Car visualizer doesn't work with me 

### ISSUE THREAD

**Issue by mustafaelnagar** [jfk94/aot_webapp#2]
Thanks for the perfect rep first.
I have downloaded it to make good use of the car visualizer like this http://carvisualizer.plus360degrees.com/threejs/ , but it doesn't work with me. 

Should I add the 3rd party of the webgl and the car parts to the code or , who to make it works well like the demo.  
Thanks in advance

**Comment by Abishek05** [comment]
Yes, It doesn't work for me either. Any pointers would be appreciated.
